server/protocols/jmap/src/test/java/org/apache/james/jmap/utils/JsoupHtmlTextExtractorTest.java

```diff
@@ -53,14 +52,14 @@ public void toPlainTextShouldRemoveSimpleHtmlTag() {
     @Test
     public void toPlainTextShouldReplaceSkipLine() {
         String html = "<p>This is an<br/>HTML text !</p>";
-        String expectedPlainText = "This is an\nHTML text !\n";
+        String expectedPlainText = "This is an HTML text !";
```

Comment:
We have a little problem then... Jsoup do not support this out of the box. We would need iterating on the structure, and detect **<p>**, **<br/>**, etc...

An innocent search *jsoup text with line breaks* end up with only results on stackoverflow...

Should I invest time in a manual DOM parsing, with text aggregation, and appending "\n" when I encounter **p** or **br**?